docs/content/connectors/mysql-cdc.md

```diff
@@ -252,6 +252,13 @@ During a snapshot operation, the connector will query each included table to pro
           <td>Integer</td>
           <td>The connection pool size.</td>
     </tr>
+    <tr>
+          <td>jdbc.properties.*</td>
+          <td>optional</td>
+          <td style="word-wrap: break-word;">20</td>
+          <td>String</td>
+          <td>Custom JDBC URL properties. Default value is 'zeroDateTimeBehavior=CONVERT_TO_NULL'.</td>
```

Comment:
please update the description, we have no default value for `custom` properties, right? 
We can say  :
Option to pass custom JDBC URL properties, user can pass custom property like `'jdbc.properties.useSSL'= 'false'`.